### PR TITLE
Remove unused Legend column of the tree in the xy-output-component

### LINKED ISSUE #579: Empty legend column in XY Charts
Each xy-chart has a legend column as last column in the tree which is currently not populated. This was done to mimic the way it was done in Eclipse Trace Compass. 

image

The column should be removed from the view because it doesn't contain any data.

### PR DESCRIPTION
**Pull request by bhufmann**
fixes #579

Signed-off-by: Bernd Hufmann <Bernd.Hufmann@ericsson.com>

### CODE REVIEW

**Review by PatrickTasse — DISMISSED** [review]

**Review by PatrickTasse — APPROVED** [review]

**Comment by ebugden** [comment]
Could this be merged? My understanding is that if the submitter is a committer we just need one more approval.

I also gave it a quick try and it looks like the code behaves as intended.

**Review by MatthewKhouzam — APPROVED** [review]

**Pull request merged**
